libr/core/cmd_write.c

```diff
@@ -94,21 +94,22 @@ static void cmd_write_op (RCore *core, const char *input) {
 	ut8 *buf;
 	int len;
 	const char* help_msg[] = {
-		"Usage:","wo[asmdxoArl24]"," [hexpairs] @ addr[!bsize]",
+		"Usage:","wo[asmdxoArl24][y]"," [hexpairs] @ addr[!bsize]",
 		"wow"," [val]", "==  write looped value (alias for 'wb')",
 		"woa"," [val]", "+=  addition (f.ex: woa 0102)",
 		"wos"," [val]", "-=  substraction",
 		"wom"," [val]", "*=  multiply",
 		"wod"," [val]", "/=  divide",
-		"woe"," [from to] [step] [wsz=1]","..  create sequence",
 		"wox"," [val]","^=  xor  (f.ex: wox 0x90)",
 		"woo"," [val]","|=  or",
 		"woA"," [val]","&=  and",
-		"woR","","random bytes (alias for 'wr $b')",
 		"wor"," [val]", ">>= shift right",
 		"wol"," [val]","<<= shift left",
 		"wo2"," [val]","2=  2 byte endian swap",
 		"wo4"," [val]", "4=  4 byte endian swap",
+		"wo*y", "", "same as their counterparts above, but using clipboard instead of [val]",
```

Comment:
No offense taken at all! I was just slightly worried that you felt offended because you started justifying radare code quality there.